modules/core/src/v2/coins/xlm.ts

```diff
@@ -747,6 +895,7 @@ export class Xlm extends BaseCoin {
           keychains = yield Bluebird.props({
             user: this.keychains().get({ id: wallet._wallet.keys[0] }),
             backup: this.keychains().get({ id: wallet._wallet.keys[1] }),
+            // TODO: why are we getting the bitgo key?
```

Comment:
i'm not sure either

Reply:
> i'm not sure either

I have a feeling we don't need it, but maybe I'm missing something? @CapnMigraine any ideas? Probably a good idea to have you review this change anyway.

Reply:
No reason, we don't need it. Please remove it.